config/blacklight.yml

```diff
@@ -1,9 +1,9 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: <%= ENV['SOLR_DEVELOPMENT_URL'] || "http://127.0.0.1:8983/solr/hydra-development" %>
```

Comment:
Style thing, but you can replace hardcoding `127.0.0.1`, with `localhost` here and in all of the other files, I believe.